src/components/asset/AssetGraphBorrow.tsx

```diff
@@ -0,0 +1,178 @@
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import { Box, IconButton, useTheme } from '@mui/material';
+import { axisClasses } from '@mui/x-charts';
+import { LineChart } from '@mui/x-charts/LineChart';
+import { useBackstop, usePool, usePoolOracle } from '../../hooks/api';
+import { toPercentage } from '../../utils/formatter';
+import { estimateEmissionsApr, estimateInterestRate } from '../../utils/math';
+import { AprDisplay } from '../common/AprDisplay';
+import { ReserveComponentProps } from '../common/ReserveComponentProps';
+import { Row } from '../common/Row';
+import { TooltipText } from '../common/TooltipText';
+
+export const AssetGraphBorrow: React.FC<ReserveComponentProps> = ({ poolId, assetId }) => {
+  const theme = useTheme();
+  const { data: pool } = usePool(poolId);
+  const { data: poolOracle } = usePoolOracle(pool);
+  const { data: backstop } = useBackstop();
+  const reserve = pool?.reserves.get(assetId);
+  const oraclePrice = poolOracle?.getPriceFloat(assetId);
+  const emissionsPerAsset = reserve?.emissionsPerYearPerBorrowedAsset();
+  const emissionApr =
+    backstop && emissionsPerAsset && emissionsPerAsset > 0 && oraclePrice
+      ? estimateEmissionsApr(emissionsPerAsset, backstop.backstopToken, oraclePrice)
+      : undefined;
+  let dataPoints: { util: number; apr: number }[] = [];
+  if (reserve) {
+    let utilizationRates = Array.from({ length: 100 }, (_, i) => i / 100);
```

Comment:
can we include the data point of 100% util on both borrow and supply graphs?

Reply:
does this also include IR modifier? We should also include the IR modifier somewhere on this page, and potentially scale the graph by the current IR modifier as well.